### PR TITLE
Show vertical line in every env.

### PR DESCRIPTION
**Pull request by andrey-star**
* Vertical line is shown in every env, not depending on current highlighted one

image

image

### CODE REVIEW

**Comment by stofee** [comment]
looking at this, I think of having the env name the same way for expanded as collapsedwould be more user friendly, the line not being at the same location is equally disturbing, as not knowing the time point was before... this change is fine, as we will need the line anyways, but we should reconsider the layout there.

**Comment by andrey-star** [comment]
> looking at this, I think of having the env name the same way for expanded as collapsedwould be more user friendly, the line not being at the same location is equally disturbing, as not knowing the time point was before... this change is fine, as we will need the line anyways, but we should reconsider the layout there.

Changed it to look like this:
image

**Comment by stofee** [comment]
nice, this looks much better, can you update the screenshot in the pr desc? fine if not any more...

File: step-release-vis/src/app/models/Data.ts [hunk 1]
```diff
@@ -7,3 +7,7 @@ import {
 export type Environment = EnvProto.AsObject;
 export type CandidateInfo = CandidateInfoProto.AsObject;
 export type Snapshot = SnapshotProto.AsObject;
+export interface Timestamp {
```
Comment (stofee) [inline step-release-vis/src/app/models/Data.ts:10]: can we use this directly from the proto instead?

**Review by stofee — APPROVED** [review]

**Review by stofee — APPROVED** [review]

File: step-release-vis/src/app/models/Data.ts (hunk 1 above)
Comment (andrey-star) [reply step-release-vis/src/app/models/Data.ts]: Done

**Pull request merged**